thirdparty/README.md

```diff
@@ -546,6 +546,17 @@ Patch `godot-node-debug-fix.patch` workarounds shadowing of Godot's Node class
 in the MSVC debugger.
 
 
+## manifold
+
+- Upstream: https://github.com/elalish/manifold
+- Version: master (12b43f50c44988cf3bc650df9fda180c8cc06163, 2024)
```

Comment:
```suggestion
- Version: git (12b43f50c44988cf3bc650df9fda180c8cc06163, 2024)
```

But if that's the commit we're tracking, it seems to be a bit older than the newly released 3.0.0, so I would suggest updating the lib to https://github.com/elalish/manifold/releases/tag/v3.0.0 and then documenting it like this:
```suggestion
- Version: 3.0.0 (5d127e57fbfb89225a8e905d0d914ccc86c139c8, 2024)
```